packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts

```diff
@@ -123,9 +123,9 @@ export const PARTITION_MAP: { [region: string]: Region } = {
 };
 
 export const CR_DEFAULT_RUNTIME_MAP: Record<Partition, string> = {
-  [Partition.Default]: 'nodejs16.x',
-  [Partition.Cn]: 'nodejs16.x',
-  [Partition.UsGov]: 'nodejs16.x',
+  [Partition.Default]: 'nodejs18.x',
+  [Partition.Cn]: 'nodejs18.x',
+  [Partition.UsGov]: 'nodejs18.x',
   [Partition.UsIso]: 'nodejs14.x',
   [Partition.UsIsoB]: 'nodejs14.x',
```

Comment:
Since we rely on the AWS SDK that Lambda builds into the runtime, either all regions have to be Node18 (SDK v3) or < Node 18 (SDK v2).

We can't have a mix of 14 and 18 using one code base.

Reply:
Thanks for the review!
So moving `UsIso` and `UsIsoB` to Node 18 would solve [this](https://github.com/aws/aws-cdk/pull/26475#issuecomment-1650492079) runtime error?

Reply:
@lpizzinidev I'm working on the runtime issue. I think we will also take over this PR and extend it to what needs to change.

Reply:
@mrgrain 👍 Let me know if you need me to apply further changes

Reply:
Thanks @lpizzinidev  If you wanted to update the iso regions to node18 as well and then your PR should be ready to go once https://github.com/aws/aws-cdk/pull/26609 is merged. 🚀